src/coreclr/jit/emitxarch.cpp

```diff
@@ -14373,29 +14373,28 @@ emitter::insExecutionCharacteristics emitter::getInsExecutionCharacteristics(ins
             if ((id->idOpSize() == EA_8BYTE))
             {
                 result.insThroughput = PERFSCORE_THROUGHPUT_52C;
-                result.insLatency    = PERFSCORE_LATENCY_62C;
+                result.insLatency += PERFSCORE_LATENCY_62C;
```

Comment:
I don't believe that the Memory latencies of the source operand (if present) necessarily sum fort these divide operations.